tests/test_simulator/test_measurements.py

```diff
@@ -245,7 +255,7 @@ def test_measurement_rng_seed(self):
         simulate(circuit, rng_seed=666)
         expected = tuple(b.value for b in circuit.bits)
 
-        for _ in range(100):
+        for _ in range(5):
             for bit in circuit.bits:
                 bit.reset()
             simulate(circuit, rng_seed=666)
```

Comment:
Won't this always produce the same outcome since it resets the seed each time `simulate` is called?

Reply:
It should, the idea behind the test is to ensure it with a high probability. With 10 qubits and 5 rounds, that comes out to a roughly 1 in 10^15 chance of it passing erroneously.